datamodel/initdb.d/03_dcsa_im_v3_0.sql

```diff
@@ -228,7 +228,8 @@ CREATE TABLE dcsa_im_v3_0.service (
     carrier_id uuid NULL REFERENCES dcsa_im_v3_0.carrier (id),
     carrier_service_code varchar(5),
     carrier_service_name varchar(50),
-    tradelane_id varchar(8) NULL REFERENCES dcsa_im_v3_0.tradelane(id)
+    tradelane_id varchar(8) NULL REFERENCES dcsa_im_v3_0.tradelane(id),
+    universal_service_reference varchar(8) NULL UNIQUE
```

Comment:
@HenrikHL and @FinnAtAsseco : I don't think this can be UNIQUE.  Logically, a `service` is a carriers definition of the service.  Per earlier discussion, multiple carriers can use the same universal service reference (when they are in a VSA) but have distinct carrier service codes for it.  I.e. Two or more rows in `service` with the same universal service reference.

Reply:
Additionally, services codes (including the universal one) can be reused / repurposed in various "exciting" commercial ways - such as but not limited to reusing the (universal/carrier) service codes for a new service after the old one has not been used for "a while" (FSVO "a while").

Reply:
I agree

Reply:
Removed the unique contraint